app/models/__init__.py

```
@@ -2,10 +2,20 @@
 
-__all__ = ["User", "Thread", "Session", "Event", "Task", "Snapshot", "AgentMetrics"]
+__all__ = [
+    "User",
+    "Thread",
+    "Session",
+    "Event",
+    "Task",
+    "Snapshot",
+    "AgentMetrics",
+    "RevokedToken",
+]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)